api/prisma/schema.prisma

```diff
@@ -9,3 +9,51 @@ datasource db {
   provider = "mysql"
   url      = env("DATABASE_URL")
 }
+
+model Business {
+  businessID    Int     @id @default(autoincrement())
+  email String  @unique
+  password String
+  businessName String
+  businessType String
+  description String
+  location String
+  professionals Professional[]
+  // reviews Review[]
+  // customers Customer[]
+  services Service[]
+}
+
+model Professional {
+  professionalID Int @id @default(autoincrement())
+  business Business @relation(fields: [professionalID], references: [businessID])
+  firstName String
+  lastName String
+  professionalService ProfessionalServices[]
+}
+
+model Service {
+  serviceID    Int     @id @default(autoincrement())
+  business Business @relation(fields: [serviceID], references: [businessID])
```

Comment:
business should be:

`business Business @relation(fields: [businessID], references: [businessID])`